plugins/android/README.md

```diff
@@ -0,0 +1,56 @@
+# List of supported targets
+
+### androidBinary
+The androidBinary target handles packaging Android application bundles into distributable APK files and publishing app artifacts. It contains Gradle tasks for generating final release builds, running ProGuard, signing the APK, versioning, etc.
+
+An androidBinary can only depend on an androidApp target. The binary encapsulates distribution logic and therefore needs to access the app configuration like flags, keys, features etc. But the app code itself should not depend on the binary. This prevents the app layer from getting coupled to low level packaging details.
+
+### androidApp
+The androidApp target is used to configure and package Android applications. It contains code, resources and configurations like feature flags, keys, debug modes etc. specific to an app.
```

Comment:
Does androidApp package Android applications? The androidBinary does, doesn't it?